packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcCreate.ts

```diff
@@ -80,7 +80,7 @@ class ForceLightningLwcCreateExecutor extends (SfdxCommandletExecutor as {
     return new SfdxCommandBuilder()
       .withDescription(nls.localize('force_lightning_lwc_create_text'))
       .withArg('force:lightning:component:create')
-      .withFlag('--type', 'web')
+      .withFlag('--type', 'lwc')
```

Comment:
This is the new flag for the CLI. This naming change aligns better with our messaging.